keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/ArtifactSummary.kt

```diff
@@ -14,9 +16,29 @@ data class ArtifactSummary(
   val versions: Set<ArtifactVersionSummary> = emptySet()
 )
 
+@JsonInclude(Include.NON_NULL)
 data class ArtifactVersionSummary(
   val version: String,
-  val environments: Set<ArtifactSummaryInEnvironment>
+  val displayName: String,
+  val environments: Set<ArtifactSummaryInEnvironment>,
+  val build: BuildMetadata? = null,
+  val git: GitMetadata? = null
+)
+
+/**
+ * todo eb: other information should go here, like a link to the jenkins build. But that needs to be done
```

Comment:
I'd envisioned this would actually be a link to the new CI view within Spinnaker. That should take care of the "scalable" part, no?

Reply:
I think all that is up for debate! All I mean is this will contain a link to _something_. Since the CI in spinnaker view might not be available for everyone, but jenkins will be, it seems reasonable to leave the comment text as is now.

I mean that we need to figure out how to get this info by querying some system, and I doubt that will be hammering jenkins/stash for it. Maybe we build a cache and populate it as new versions come it? That kind of thing. We will address the design of that later though.

Reply:
> Since the CI in spinnaker view might not be available for everyone, but jenkins will be, it seems reasonable to leave the comment text as is now.

Topic for a separate discussion for sure, but I was surprised by this. I thought the whole point of the CI view was to abstract Jenkins (and other CI systems). Based on conversation with @erikmunson and @gal-yardeni recently, it seems everyone was in agreement that that's where those links would point, hence my comment. But let's chat offline. 😃

Reply:
@luispollo CI in Spinnaker will be available for rocket users first. We do have a plan to make it generic (and to open source it), but it will take a while to do so. We could find how to link between the two, but it still very early in the process.